demo/src/main/java/com/orange/ods/demo/ui/components/radiobuttons/ComponentRadioButtons.kt

```diff
@@ -44,28 +44,17 @@ fun ComponentRadioButtons() {
                 .verticalScroll(rememberScrollState())
                 .padding(bottom = dimensionResource(id = R.dimen.spacing_m))
         ) {
-            val selectedRadio = remember { mutableStateOf(R.string.component_element_item1) }
+            val recipes = LocalRecipes.current.take(3)
+            val selectedRadio = remember { mutableStateOf(recipes.firstOrNull()?.title) }
```

Comment:
I think this should be a `rememberSaveable` cause we lost the selection on screen orientation change.